scripts/serve/serve-cli.js

```diff
@@ -9,7 +9,7 @@ commander
   .version(packageJson.version)
   .option('--config <path>', 'The webpack config to serve.', undefined)
   .option('--host <s>', 'Sets the host that the server will listen on. eg. \'10.10.10.1\'', '0.0.0.0')
-  .option('--port <n>', 'The port the server should listen on.', parseInt, 8080)
+  .option('--port <n>', 'The port the server should listen on.', Number(parseInt), 8080)
```

Comment:
This is not an issue for serve-static because we do not define a default value, however I could make this change there as well.